bot/exts/moderation/infraction/management.py

```diff
@@ -256,15 +257,69 @@ async def search_reason(self, ctx: Context, reason: str) -> None:
             'bot/infractions/expanded',
             params={'search': reason}
         )
+
+        formatted_infraction_count = self.format_infraction_count(len(infraction_list))
+        embed = discord.Embed(
+            title=f"Infractions matching `{reason}` ({formatted_infraction_count} total)",
+            colour=discord.Colour.orange()
+        )
+        await self.send_infraction_list(ctx, embed, infraction_list)
+
+    # endregion
+    # region: Search for infractions by given actor
+    @infraction_group.command(name="by", aliases=("b",))
```

Comment:
Shouldn't there be an empty line here?
```suggestion
    # region: Search for infractions by given actor

    @infraction_group.command(name="by", aliases=("b",))
```